bigquery_etl/view/generate_stable_views.py

```diff
@@ -220,7 +223,7 @@ def main():
                     continue
                 schemas.append(
                     SchemaFile(
-                        schema_id=schema["$schema"],
+                        schema_id=schema.get("$id", ""),
```

Comment:
This line was the real problem. We were fetching `$schema` rather than `$id` so no table ever matched based on `schema_id`.